tarun_kumar_PostgreAssignment/src/controller/student-controller.ts

```diff
@@ -0,0 +1,61 @@
+import { Context } from 'vm';
+import { v4 as uuidv4 } from 'uuid';
+
+import * as studentService from '../services/student-service';
+
+
+export async function addStudent(ctx: Context) {
+    try {
+        const requestData = ctx.request.body;
+
+        const id: string = uuidv4();
+        const name: string = requestData.name.trim();
```

Comment:
we are not validating request body, like age is number, name is present and its string. Can you check https://joi.dev/api/?v=17.4.0 to do validation

Reply:
done